rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -37,6 +42,22 @@ THE SOFTWARE.
 #include "node_resize.h"
 #include "rocal_api.h"
 
+#ifdef ROCAL_AUDIO
+std::tuple<unsigned, unsigned>
+evaluate_audio_data_set(StorageType storage_type, DecoderType decoder_type,
+                        const std::string& source_path, const std::string& json_path) {
```

Comment:
change the json path to file_list_path here

Reply:
Done